src/document/utils.js

```diff
@@ -161,16 +161,20 @@ function mapDoc(doc, cb) {
 function findInDoc(doc, fn, defaultValue) {
   let result = defaultValue;
   let hasStopped = false;
+
   function findInDocOnEnterFn(doc) {
+    if (hasStopped) {
+      // Stop processing rest docs
+      return false;
```

Comment:
Doesn't `return false` stop the further processing immediately? Is the `hasStopped` variable really needed?

Reply:
See https://github.com/prettier/prettier/pull/13349#discussion_r952138588

Reply:
I thought it does, but it's not. It will keep processing the siblings.

Reply:
Ah, okay. I see it's needed.